chapters/equations.tex

```diff
@@ -858,6 +860,39 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \end{lstlisting}
 \end{example}
 
+\begin{example}
+Resettable continuous time controller initialized either in steady-state or by providing a \lstinline!start! value for state \lstinline!y!:
```

Comment:
This looks like the exact same introductory sentence as used in an existing example.  If that's the case, the two should be diversified somehow.

Also, it looks like we should be consistent and the hyphenated form _continuous-time_ (in both examples):
```suggestion
Resettable continuous-time controller initialized either in steady-state or by providing a \lstinline!start! value for state \lstinline!y!:
```